applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.h

```diff
@@ -101,6 +101,10 @@ int ble_hci_vsc_set_radio_high_pwr_mode(bool high_power_mode);
  * @brief Set Bluetooth MAC device address
  * @param bd_addr	Bluetooth MAC device address
  *
+ * @note	This can be used to set a public address for the device
+ *		Note that bt_setup_public_id_addr(void) should be called
```

Comment:
Remove "void"?

Reply:
I included it there to indicate that it is a function that didn't require any input, so it was deliberate.